catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegateTest.java

```diff
@@ -584,10 +564,7 @@ public void testPropertyIsEqualToLong() throws JAXBException, SAXException, IOEx
   public void testPropertyIsEqualToFloat() throws JAXBException, SAXException, IOException {
     mockProps.add(MOCK_PROPERTY);
```

Comment:
❓ Why isn't this using `whenProperties()`? Same with L579 below

Reply:
I just refactored these tests so they are now reusing the setup code.